uaclient/cli.py

```diff
@@ -206,6 +202,10 @@ def attach_parser(parser):
     """Build or extend an arg parser for attach subcommand."""
     parser.usage = USAGE_TMPL.format(name=NAME, command="attach <token>")
     parser.prog = "attach"
+    parser.description = (
+        "Attach this machine to Ubuntu Advantage support with a token obtained"
```

Comment:
apologies, I must have missed this one. Again I think that "support" is an overloaded word and so try to avoid it if we can. I think we can just drop it.